spec/reek/smells/too_many_statements_spec.rb

```diff
@@ -31,34 +26,12 @@ def process_singleton_method(src)
   end
 
   it 'should not report initialize' do
-    src = '
+    src = <<-EOS
       def initialize(arga)
         alf = f(1); @bet = 2; @cut = 3; @dit = 4; @emp = 5; @fry = 6
       end
-    '
-    expect(src).not_to reek_of(TooManyStatements)
-  end
-
-  it 'should only report a long method once' do
-    src = <<-EOS
-      def standard_entries(rbconfig)
-        @abc = rbconfig
-        rubypath = File.join(@abc['bindir'], @abcf['ruby_install_name'] + cff['EXEEXT'])
-        major = yyy['MAJOR'].to_i
-        minor = zzz['MINOR'].to_i
-        teeny = ccc['TEENY'].to_i
-        version = ""
-        if c['rubylibdir']
-          @libruby         = "/lib/ruby"
-          @librubyver      = "/lib/ruby/"
-          @librubyverarch  = "/lib/ruby/"
-          @siteruby        = "lib/ruby/version/site_ruby"
-          @siterubyver     = siteruby
-          @siterubyverarch = "$siterubyver/['arch']}"
-        end
-      end
     EOS
-    expect(src).to reek_only_of(:TooManyStatements)
```

Comment:
Yeah I can't figure out where this test came from in the first place. There must have been some issue that triggered this, perhaps some other smell also being reported? Seems reasonable to get rid of it.